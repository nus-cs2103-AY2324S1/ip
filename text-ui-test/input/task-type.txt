todo buy grocery
event Orbital splashdown /from tday 6pm /to tday 9pm
deadline finish this project /by Thurs 2359
event GDG-workshop /from tmr 7.30pm /to tmr 9.30pm
list
mark 10
mark 4
list
deadline CS2109S PS0
deadline CS2109S PS0 /by 26 Aug
event cchess training /to 9pm Mon /from 7pm Mon
event cchess training /from 7pm Mon /t 9pm
event cchess training /from 7pm Mon /to 9pm
list
bye