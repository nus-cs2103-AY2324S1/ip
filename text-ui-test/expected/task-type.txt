____________________________________________________________
Loading data from hard disk ...
Done loading.
____________________________________________________________
Hello from Quack-NKN
What can I do for you?
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] buy grocery
Now you have 1 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[E][ ] Orbital splashdown (from: tday 6pm to: tday 9pm)
Now you have 2 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[D][ ] finish this project (by: Thurs 2359)
Now you have 3 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[E][ ] GDG-workshop (from: tmr 7.30pm to: tmr 9.30pm)
Now you have 4 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] buy grocery
2. [E][ ] Orbital splashdown (from: tday 6pm to: tday 9pm)
3. [D][ ] finish this project (by: Thurs 2359)
4. [E][ ] GDG-workshop (from: tmr 7.30pm to: tmr 9.30pm)
____________________________________________________________
In: ____________________________________________________________
Quack, the task number you provide does not exist!
____________________________________________________________
In: ____________________________________________________________
Nice! I've marked this task as done:
[E][X] GDG-workshop (from: tmr 7.30pm to: tmr 9.30pm)
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] buy grocery
2. [E][ ] Orbital splashdown (from: tday 6pm to: tday 9pm)
3. [D][ ] finish this project (by: Thurs 2359)
4. [E][X] GDG-workshop (from: tmr 7.30pm to: tmr 9.30pm)
____________________________________________________________
In: ____________________________________________________________
Quack, keyword '/by' not found.It must be present for me to mark the deadline time!
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[D][ ] CS2109S PS0 (by: 26 Aug)
Now you have 5 in the list.
____________________________________________________________
In: ____________________________________________________________
Quack, keyword '/to' not found. It must be present after the '/from' keyword for me to mark the end time!
____________________________________________________________
In: ____________________________________________________________
Quack, keyword '/to' not found. It must be present after the '/from' keyword for me to mark the end time!
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[E][ ] cchess training (from: 7pm Mon to: 9pm)
Now you have 6 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] buy grocery
2. [E][ ] Orbital splashdown (from: tday 6pm to: tday 9pm)
3. [D][ ] finish this project (by: Thurs 2359)
4. [E][X] GDG-workshop (from: tmr 7.30pm to: tmr 9.30pm)
5. [D][ ] CS2109S PS0 (by: 26 Aug)
6. [E][ ] cchess training (from: 7pm Mon to: 9pm)
____________________________________________________________
In: ____________________________________________________________
Bye. Hope to see you again soon!
____________________________________________________________